webapp/src/ts/services/contact-summary.service.ts

```diff
@@ -89,8 +99,10 @@ export class ContactSummaryService {
       uhcInterval: this.uhcStatsService.getUHCInterval(this.visitCountSettings)
     };
 
+    const chtScriptApi = this.chtScriptApiService.getV1Api();
+
     try {
-      const summary = generatorFunction(contact, reports || [], lineage || [], uhcStats, targetDoc);
+      const summary = generatorFunction(contact, reports || [], lineage || [], uhcStats, chtScriptApi, targetDoc);
```

Comment:
Everywhere we expose the API it needs to be versioned, eg:

```suggestion
      const summary = generatorFunction(contact, reports || [], lineage || [], uhcStats, { v1: chtScriptApi }, targetDoc);
```

Reply:
The service is already returning the api in a version obj, I rather have the versions managed in 1 place than in multiple places 🤔

Reply:
Ahh ok! Then the name is wrong. Rename it to just `this.chtScriptApiService.get();` because one day it'll return `{ v1: {}, v2: {} }`

Reply:
oh right! [facepalm] I'm going to rename it

Reply:
Fixed